db/forms/kpi/gbv_assessment.rb

```diff
@@ -0,0 +1,28 @@
+gbv_assessment_fields = [
+  Field.new({"name" => "completed_asssessment",
+    "mobile_visible" => false,
+    "type" => "tick_box",
+    "display_name_en" => "Completed assessment",
+    "editable" => false,
```

Comment:
All of your fields are set as "editable" false.  This prevents the field from being changed in the Form Editor.   Is that what you intended?